lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -79,19 +88,34 @@ newNetworkLayer url = do
     return $ mkNetworkLayer $ mkJormungandrLayer mgr url
 
 -- | Wrap a Jormungandr client into a 'NetworkLayer' common interface.
-mkNetworkLayer :: Monad m => JormungandrLayer m -> NetworkLayer t m
+mkNetworkLayer
+    :: Monad m
+    => JormungandrLayer m
+    -> NetworkLayer t m
 mkNetworkLayer j = NetworkLayer
     { networkTip = do
         t <- (getTipId j) `mappingError`
             ErrNetworkTipNetworkUnreachable
         b <- (getBlock j t) `mappingError` \case
-            ErrGetBlockNotFound (BlockId _) ->
+            ErrGetBlockNotFound _ ->
                 ErrNetworkTipNotFound
             ErrGetBlockNetworkUnreachable e ->
                 ErrNetworkTipNetworkUnreachable e
         return $ header b
 
-    , nextBlocks = error "nextBlocks to be implemented"
+    , nextBlocks = \tip -> do
+        let count = 10000
+        -- Get the descendants of the tip's /parent/.
+        -- The first descendant is therefore the current tip itself. We need to
+        -- skip it. Hence the 'tail'.
+        ids <- tail <$> getDescendantIds j (prevBlockHash tip) count
```

Comment:
> tail :: Extract the elements after the head of a list, which **must be non-empty**.

We probably need to have a `safeTail` here that returns an empty list of the list is already empty :). Gotta love Haskell's type safety.

Reply:
Added 

```haskell
    tailOrEmpty [] = []
    tailOrEmpty (_:xs) = xs
```

but if the tip-parent has no descendant, that seems pretty bad 🤔 🤷‍♂